src/Sylius/Behat/Context/Api/Shop/ProductContext.php

```diff
@@ -44,6 +46,36 @@ public function iOpenProductPage(ProductInterface $product): void
         $this->client->show($product->getCode());
     }
 
+    /**
+     * @When /^I browse products from (taxon "([^"]+)")$/
+     */
+    public function iCheckListOfProductsForTaxon(TaxonInterface $taxon): void
+    {
+        $this->client->index();
+        $this->client->addFilter('productTaxons.taxon.code', $taxon->getCode());
+        $this->client->filter();
+    }
+
+    /**
+     * @Then I should see the product price :productPrice
+     */
+    public function iShouldSeeTheProductPrice(string $productPrice): void
+    {
+        $price = (int) filter_var($productPrice, FILTER_SANITIZE_NUMBER_INT);
```

Comment:
is it necessary? it should be transform by `sylius.behat.context.transform.lexical` to int